web/Views/Property/ViewProperty.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@model PropertyViewModel
+
+@{
+    ViewData["Title"] = "View Property";
+}
+
+<div class="container">
+    <div class="row mb-3">
+        <h3>View property</h3>
+    </div>
+    <div class="row mb-3 ms-0">
+        <div class="col-7 border p-2">
+            <p>@Model.Address.AddressLine1</p>
+            <p>Current rental: TODO</p>
+            <p>Looking for tenant: TODO</p>
```

Comment:
I would remove the 'TODO' and leave it blank, then you could raise a couple of tickets in the 'Property management' epic to create these sections